media-sound/sequencer64/sequencer64-9999.ebuild

```diff
@@ -29,10 +29,6 @@ RDEPEND="dev-cpp/gtkmm:2.4
 
 DEPEND="${RDEPEND}"
 
-PATCHES=(
```

Comment:
I would say this belongs to another commit, but I'm not to anal about it.
Do you know how to split commits ?